modules/core/src/main/scala/org/scalasteward/core/update/UpdateAlg.scala

```diff
@@ -79,13 +50,39 @@ final class UpdateAlg[F[_]](implicit
     updates.flatMap(filterAlg.localFilterMany(repoConfig, _))
```

Comment:
You could inline `updates` and get rid of the curly braces, but wouldn't save a line